app/services/ExerciseService.ts

```diff
@@ -0,0 +1,115 @@
+import { marked } from 'marked'
+import NotFoundException from '../exceptions/NotFoundException';
+import fs from "fs";
+
+export const ExericseService = {
+  /**
+* Gets the content from the file
+*
+* @param {string} Exercise_ID id for an exercise
+* @return {string} the content being returned
+* @throws {NotFoundException} File is Not found exception handler
+*/
+  getContent(Exercise_ID: string): string {
+    const fileLocation = "exercises/" + Exercise_ID + ".md";
+    try {
+      const fileContent = fs.readFileSync(fileLocation, "utf8");
+      const form = marked.parse(fileContent)
+      var words = form.split("<hr>")
+      return words[0] + words[2]
```

Comment:
This is a pretty perfect area to get in the habit of commenting. At the moment I'm not sure why we're splitting by "<hr>", and in a year from now you might not remember exactly either. Nor what the contents of words[0] and words[2] are. 

Can you add some descriptive comments to shed light on those two things?